app/src/main/AndroidManifest.xml

```diff
@@ -74,9 +74,6 @@
                 <data android:mimeType="application/vnd.wap.xhtml+xml" />
             </intent-filter>
         </activity>
-        <activity
-            android:name="fr.gaulupeau.apps.Poche.ui.AddActivity"
-            android:label="@string/activity_add_label" />
```

Comment:
`@string/activity_add_label` is not used anymore. It would be a good idea to rename it to something like `@string/dialog_add_label` and use it as the dialog title (instead of `nav_add`).